project_redss/lib/channels.py

```diff
@@ -64,19 +86,31 @@ def set_channel_keys(cls, user, data, time_key):
             timestamp = isoparse(td[time_key])
 
             channel_dict = dict()
+
+            # Set channel ranges
             time_range_matches = 0
-            for key, ranges in cls.RANGES.items():
+            for key, ranges in cls.CHANNEL_RANGES.items():
                 if cls.timestamp_is_in_ranges(timestamp, ranges):
                     time_range_matches += 1
                     channel_dict[key] = Codes.TRUE
                 else:
                     channel_dict[key] = Codes.FALSE
 
-            assert time_range_matches <= 1, f"Time '{td[time_key]}' matches multiple time ranges"
-
+            # Set time as NON_LOGICAL if it doesn't fall in range of the **sms ad/radio promo/radio_show**
```

Comment:
- I thought we were using 'Not Internally Consistent' or something similar now?
- Should we create another control code 'Out of range'? Someone talking to us outside our data window isn't necessarily not logical

Reply:
This isn't a control code, but agreed that it could much more descriptive. @ParthaMoman